.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "gml"]
+	path = gml
+	url = https://github.com/ilmola/gml.git
```

Comment:
Would be great to be able to use this from a system package as well.
Currently there's no release of this header-only library though.

Regarding submodules: Please keep in mind, that if you do a `git tag`, github will not include submodules in your tarball!

Reply:
> Currently there's no release of this header-only library

Yes, it's not very well known. But I liked it because it is very small. Most other libraries I found had a million other features that I don't need.

If you know a better option, just let me know, we can easily switch. I deliberately didn't use GML types in any API functions.

> github will not include submodules in your tarball!

I know. But Github's tarball is quite useless anyway, because it doesn't have the `./configure` in it. We ship our separate tarballs anyway.